Cargo.lock

```diff
@@ -3255,24 +3923,44 @@ dependencies = [
  "serde_json",
  "sway-core",
  "sway-fmt",
- "sway-types",
- "sway-utils",
+ "sway-types 0.10.3",
+ "sway-utils 0.10.3",
  "tokio",
  "tower",
  "tower-lsp",
 ]
 
+[[package]]
+name = "sway-types"
+version = "0.8.0"
```

Comment:
Ahhhh, still a bit of the old cyclic dependency between `fuels-rs` an `sway` left.